src/generated/ScenarioBased.jl

```diff
@@ -2,31 +2,35 @@
 This file is auto-generated. Do not edit.
 =#
 """
-    mutable struct ScenarioBased <: Forecast
+    mutable struct ScenarioBased <: TimeSeriesData
```

Comment:
We can rename ScenatioBased to just Scenarios. @claytonpbarrows what do you think? I find awkward `ScenarioBased` time series.

Reply:
I also find it to be awkward. But I don't love `Scenarios` either. Also, with `TimeSeriesData` do we even need `ScenarioBased` now that we have both `label` and `accessor`? If the answer is yes, then I'm fine with the proposal.